pkg/machines/components/memorySelectRow.jsx

```diff
@@ -41,7 +41,8 @@ class MemorySelectRow extends React.Component {
     }
 
     render() {
-        const { id, value, minValue, maxValue, initialUnit, onValueChange, onUnitChange, readOnly } = this.props;
+        const { id, value, minValue, maxValue, onValueChange, onUnitChange, readOnly } = this.props;
```

Comment:
Can't we just always use the string instead of having this logic here? I think it's more straightforward.

Reply:
That's also an option. I just wanted MemorySelect be more failproof for this case. Helper funciton convertToUnit also accounts for parameters being both object or string.

Reply:
Fixed